R/slurmCache.R

```diff
@@ -1,129 +0,0 @@
-#Functions for SLURM binding within R... wow!
```

Comment:
was this where `slurmParams` was documented before? see [`check()`](https://github.com/databio/simpleCache/pull/5#issuecomment-318659752) results